doc/interfaces/reconstruction_flow.rst

```diff
@@ -0,0 +1,269 @@
+underlying fiber distribution.
+
+In this way, the reconstruction of the fiber orientation distribution function
+(fODF) in CSD involves two steps:
+
+* Estimation of the fiber response function
+* Use the response function to reconstruct the fODF
+
+We are using ``stanford_hardi`` dataset. You can also use your own data.
+
+Create the directories in which to save the peaks volume (e.g.: ``recons_csd_output``)
+and mask file (e.g.: ``stanford_hardi_mask``)::
+
+    mkdir recons_csd_output stanford_hardi_mask
+
+The workflow for the CSD reconstruction method requires the paths to the
+diffusion input file, b-values file, b-vectors file and mask file. The optional
+arguments can also be provided. In this case, we will specify FA threshold for
+calculating the response function (``fa_thr``), spherical harmonics order used
+in the CSA fit (``sh_order``), whether to use parallelization in peak-finding
+during the calibration procedure or not (``parallel``), and the output
+directory (``out_dir``).
+
+To get the mask file, we will run the following command::
+
+    dipy_median_otsu data/stanford_hardi/HARDI150.nii.gz --vol_idx 3 4 5 6 --out_dir "stanford_hardi_mask"
+
+Then, to perform the CSD reconstruction we will run the command as::
+
+    dipy_fit_csd data/stanford_hardi/HARDI150.nii.gz data/stanford_hardi/HARDI150.bval data/stanford_hardi/HARDI150.bvec stanford_hardi_mask/brain_mask.nii.gz --fa_thr 0.9 --sh_order 9 --parallel True --out_dir "recons_csd_output"
+
+This command will save the CSD metrics to the specified output directory.
+
+----------------------------------
+Mean Apparent Propagator (MAP)-MRI
+----------------------------------
+
+The MAP-MRI basis allows for the computation of directional indices, such as
+the Return To the Axis Probability (RTAP), the Return To the Plane Probability
+(RTPP), and the parallel and perpendicular Non-Gaussianity.
+
+The estimation of analytical Orientation Distribution Function (ODF) and a
+variety of scalar indices from noisy DWIs requires that the fitting of the ]
+MAPMRI basis is regularized.
+
+
+------------------------
+Diffusion Tensor Imaging
+------------------------
+
+The diffusion tensor model is a model that describes the diffusion within a
+voxel. First proposed by Basser and colleagues [Basser1994]_, it has been very
+influential in demonstrating the utility of diffusion MRI in characterizing the
+micro-structure of white matter tissue and of the biophysical properties of
+tissue, inferred from local diffusion properties and it is still very commonly
+used.
+
+The diffusion tensor models the diffusion signal as:
+
+.. math::
+
+    \frac{S(\mathbf{g}, b)}{S_0} = e^{-b\mathbf{g}^T \mathbf{D} \mathbf{g}}
+
+Where $\mathbf{g}$ is a unit vector in 3 space indicating the direction of
+measurement and b are the parameters of measurement, such as the strength and
+duration of diffusion-weighting gradient. $S(\mathbf{g}, b)$ is the
```

Comment:
(...)  of the diffusion-weighting gradients.  $S(\mathbf{g}, b)$ is the measures diffusion-weighted signal and (...)

Reply:
Reworded in 4ce067c